src/NodeAutoCompleteViewExtension/NodeAutoCompleteViewExtension.cs

```diff
@@ -153,5 +161,34 @@ internal void ShowClusterNodeAutocompleteResults(MLNodeClusterAutoCompletionResp
 
             DependencyView.MainItems.ItemsSource = nodeAutocompleteClusters;
         }
+
+        private static NodeAutoCompleteBarViewModel nodeAutoCompleteBarViewModel;
+
+        private void OnNodeAutoCompleteBarRequested(Window parentWindow, ViewModelBase viewModelBase)
+        {
+            PortViewModel portViewModel = viewModelBase as PortViewModel;
+            if (parentWindow is null || portViewModel is null)
+            {
+                return;
+            }
+            
+            if (nodeAutoCompleteBarViewModel is null)
+            {
+                DynamoViewModel dynamoViewModel = portViewModel?.NodeViewModel?.WorkspaceViewModel?.DynamoViewModel;
+                nodeAutoCompleteBarViewModel = new NodeAutoCompleteBarViewModel(dynamoViewModel);
+            }
+
+            if (nodeAutoCompleteBarViewModel.PortViewModel != null)
+            {
+                nodeAutoCompleteBarViewModel.PortViewModel.Highlight = Visibility.Collapsed;
+            }
+
+            nodeAutoCompleteBarViewModel.PortViewModel = portViewModel;
+
+            //TODO : I think we can reuse the window too.
+            var nnodeAutoCompleteBarWindow = new NodeAutoCompleteBarView(parentWindow, nodeAutoCompleteBarViewModel);
```

Comment:
Maybe the double `nn` was a typo we missed?

Reply:
Yes